frontend/app/src/screens/ipam/ipam-tree/utils.ts

```diff
@@ -52,6 +55,7 @@ export const formatIPPrefixResponseForTreeView = (data: PrefixData): TreeItemPro
     isBranch: node.children.count > 0,
     metadata: {
       kind: node.__typename,
+      descendants: node.descendants.count,
```

Comment:
How about renaming it to something like descendantsTotal ? Just to make it explicit that it contains a count number